controller/src/main/java/io/pravega/controller/server/SegmentHelper.java

```diff
@@ -167,41 +168,46 @@ void setTimeout(Duration duration) {
         final long requestId = connection.getFlow().asLong();
         Pair<Byte, Integer> extracted = extractFromPolicy(policy);
 
-        return sendRequest(connection, requestId, new WireCommands.CreateSegment(requestId, qualifiedStreamSegmentName,
+        return sendRequest(connection, clientRequestId, new WireCommands.CreateSegment(requestId, qualifiedStreamSegmentName,
```

Comment:
We seem to have replaced "FlowId" with clientRequestId. Does this also work for Segment Store?

Reply:
we se the flowId inside the wirecommand.
the client requestid here is something that this api received and captures the request context.. 
its simply used for logging purpose

Reply:
I see the _requestId_ being logged by processAndRethrowException(). It would be good to also add it to this log inside sentRequest():
```
log.debug("Sending request: {}", request);
```

Reply:
we are logging it inside the sendRequest method too. its just that its at trace and not debug level. 

https://github.com/pravega/pravega/blob/3ea93f2994e42b3f90b1922eb055e8ba80f03408/controller/src/main/java/io/pravega/controller/server/SegmentHelper.java#L741